seatunnel-engine/seatunnel-engine-server/src/main/java/org/apache/seatunnel/engine/server/NodeExtension.java

```diff
@@ -77,6 +78,7 @@ public TextCommandService createTextCommandService() {
             {
                 register(HTTP_GET, new Log4j2HttpGetCommandProcessor(this));
                 register(HTTP_POST, new Log4j2HttpPostCommandProcessor(this));
+                register(HTTP_GET, new RestHttpGetCommandProcessor(this));
```

Comment:
Please test `Log4j2HttpGetCommandProcessor` is normal

Reply:
@liugddx

Reply:
http://localhost:5801/hazelcast/rest/log-level

![image](https://user-images.githubusercontent.com/48236177/218623216-19464923-e1b5-4248-9186-4c668b11b448.png)